lib/syntax_tree/translation/parser.rb

```diff
@@ -506,13 +506,13 @@ def visit_block_var(node)
             )
           end
 
-        params = node.params
```

Comment:
All the changes in this method are an attempt to combine the `visit_block_var` and `visit_lambda_var` functionality, which actually aren't that far apart but were written slightly differently.